MetaMorpheus/EngineLayer/DIA/PrecursorFragmentGrouping/PfGroupingEngine.cs

```diff
@@ -0,0 +1,16 @@
+﻿using MassSpectrometry;
+using System.Collections.Generic;
+
+namespace EngineLayer.DIA
+{
+    /// <summary>
+    /// PfGroupingEngine defines the process of grouping precursor and fragment XICs into PrecursorFragmentsGroup objects. 
+    /// It should have a method that returns all PrecursorFragmentsGroup objects that can be found in a given set of precursor and fragment XICs.
+    /// <summary>
+    public abstract class PfGroupingEngine
```

Comment:
This should be an interface, as there are no methods implemented so no shared functionality